yellowbrick/cluster/elbow.py

```diff
@@ -271,8 +282,10 @@ def draw(self):
         Draw the elbow curve for the specified scores and values of K.
         """
         # Plot the silhouette score against k
-        self.ax.plot(self.k_values_, self.k_scores_, marker="D", label="score")
-
+        self.ax.plot(self.k_values_, self.k_scores_, marker="D")
+        if self.knee:
+            self.ax.axvline(self.knee_value,c='black',linestyle='--')
+            self.ax.legend(['Score={}'.format(round(self.score,3)),'Optimal k={}'.format(self.knee_value)],loc='best')
```

Comment:
To create a text-only legend, you could use [`manual_legend`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/draw.py#L29). However, I don't think that's necessary, instead you could do:

```python
elbow_label = "$elbow at k={}, score={0.2f}$".format(self.elbow_value_, self.elbow_score_)"
self.ax.axvline(self.elbow_value_, c=LINE_COLOR, linestyle="--", label=elbow_label)
```

Then later, in `finalize()` you can simply call `self.ax.legend(loc='best')` (which is the best place to put the legend drawing, or the manual legend drawing if you decide to go that route).

Reply:
Note also the `$` in the label - that renders the label as latexmath, which might be nicer, it might not be though.

Reply:
I'll definitely go deeper into this. And would do the changes as suggested. Thanks for great info.